src/main/proto/marathon.proto

```diff
@@ -141,6 +141,22 @@ message MarathonTask {
     repeated string local_volume_ids = 1;
     required State state = 2;
   }
+  enum MarathonTaskStatus {
```

Comment:
for example, what's the difference between Gone and Lost. would be good to spell that out